shared/oae/bundles/ui/default.properties

```diff
@@ -498,6 +498,9 @@ USERNAME = Username
 USERNAME_COLON = Username:
 USER_MANAGEMENT = User management
 VIDEO = Video
+VISIBILE_ONLY_TO_SPECIFIED_USERS = Visible only to specified users
```

Comment:
I think it would be good if this could re-use the visibility messages from the manageaccess for the various entitites:

*Files*

```
FILE_PRIVATE_DESCRIPTION
FILE_LOGGEDIN_DESCRIPTION
FILE_PUBLIC_DESCRIPTION
```

*Collaborative documents*

```
DOCUMENT_PRIVATE_DESCRIPTION
DOCUMENT_LOGGEDIN_DESCRIPTION
DOCUMENT_PUBLIC_DESCRIPTION
```

*Links*

```
LINK_PRIVATE_DESCRIPTION
LINK_LOGGEDIN_DESCRIPTION
LINK_PUBLIC_DESCRIPTION
```

*Groups*

```
GROUP_PRIVATE_DESCRIPTION_PRESENT
GROUP_LOGGEDIN_DESCRIPTION
GROUP_PUBLIC_DESCRIPTION_PRESENT
```

*Discussions*

```
DISCUSSION_PRIVATE_DESCRIPTION
DISCUSSION_LOGGEDIN_DESCRIPTION
DISCUSSION_PUBLIC_DESCRIPTION
```

*Folders*

```
FOLDER_PRIVATE_DESCRIPTION
FOLDER_LOGGEDIN_DESCRIPTION
FOLDER_PUBLIC_DESCRIPTION
```